lib/new-flatfile.ts

```diff
@@ -3,8 +3,23 @@ import { ListWorkbooksRequest } from "@flatfile/api/api";
 import { getSpaceConfig } from "./flatfile";
 
 // TODO: Need to take in per-workflow API key here
-const flatfileClient = () => {
-  const token = process.env.FLATFILE_API_KEY;
+const flatfileClient = (flowName: string) => {
+  const onboarding = process.env.ONBOARDING_FLATFILE_API_KEY;
+  const embedded = process.env.FILEFEED_FLATFILE_API_KEY;
+  const filefeed = process.env.EMBEDDED_FLATFILE_API_KEY;
+  const dynamic = process.env.DYNAMIC_FLATFILE_API_KEY;
+  let token;
+
+  switch (flowName) {
+    case "onboarding":
+      token = onboarding;
+    case "embedded":
+      token = embedded;
+    case "filefeed":
+      token = filefeed;
+    case "dynamic":
+      token = dynamic;
+  }
 
   if (!token) {
     throw new Error("No FLATFILE_API_KEY ENV");
```

Comment:
let's add the flowname to this error here too so we can debug easier

Reply:
edited